odoo/addons/base/tests/test_ir_mail_server_smtpd.py

```diff
@@ -29,6 +31,11 @@
 _openssl = shutil.which('openssl')
 _logger = logging.getLogger(__name__)
 
+if getenv('ODOO_RUNBOT') and not _openssl:
+    _logger.warning("detected runbot environment but openssl but found in PATH, TestIrMailServerSMTPD will be skipped")
```

Comment:
but ... but ...

Reply:
@MiquelRForgeFlow yes first time we do something like this... Can you elaborate?